### ISSUE TITLE
ini_set('session.cookie_lifetime', 0);

### ISSUE THREAD

**Issue by mfurlend** [stefangabos/Zebra_Session#5]
On lines 235-237 you have this code:

```
// make sure session cookies never expire so that session lifetime
// will depend only on the value of $session_lifetime
ini_set('session.cookie_lifetime', 0);
```

That statement does not make "session cookies never expire." It makes the cookies expire when the browser is closed. In my case this caused authenticated users to be logged off upon termination of the session/browser.

Is this a bug or the intended behavior?

**Comment by jmathai** [comment]
Agree with @mfurlend. To use this (wonderful) library we need to fork and change that line of code. Even if we have `session.cookie_lifetime` set to something longer in `php.ini` this class will override it.

I don't think it's a valid assumption that site owners typically want sessions to end when a browser is closed.

**Comment by stefangabos** [comment]
This is now fixed by [a01bdee](https://github.com/stefangabos/Zebra_Session/commit/a01bdee6de54e8b777acd424fbc5e815b1bbc9b9)
